### DISCUSSION TITLE
Allure report enviroment - doubts

### DISCUSSION THREAD

**Discussion by motovmp1** [webdriverio/webdriverio#6975] [Q&A]
Hello friends!

describe('ecommerce login', ()=>{
    it('login page teste', ()=>    {
        allureReporter.addSeverity('Critical')
        allureReporter.addEnvironment("Enviroment", "Dev")
        allureReporter.addEnvironment("Browser: ", "Chrome")
        allureReporter.addEnvironment("Chrome: ", "91.0.0.1")

how to write in one line the enviroment?

Thanks

**Comment by portyu9** [comment]
Hi @motovmp1 

For current `@wdio/allure-reporter`, don't use `addEnvironment()` anymore. It is deprecated and no longer works.

Define the environment values in the reporter configuration using `reportedEnvironmentVars`:

```js
export const config = {
    // ...

    reporters: [[
        'allure',
        {
            outputDir: 'allure-results',

            reportedEnvironmentVars: {
                Environment: 'Dev',
                Browser: 'Chrome',
                BrowserVersion: '91.0.0.1'
            }
        }
    ]]
}
```

If by "one line" you literally mean you want all of that displayed as **one environment entry**, you can instead do:

```js
reportedEnvironmentVars: {
    Environment: 'Dev | Browser: Chrome 91.0.0.1'
}
```

Also, severity values are documented in lowercase, so I would use:

```js
allureReporter.addSeverity('critical')
```

rather than:

```js
allureReporter.addSeverity('Critical')
```

The supported severity values are:

```text
blocker
critical
normal
minor
trivial
```

So the main change since this question was posted is that environment information belongs in `reportedEnvironmentVars`, not repeated `addEnvironment()` calls.

Please mark this answer as accepted if it helped, thank you.
